pkg/apis/autoscaling/types.go

```diff
@@ -95,6 +95,90 @@ type HorizontalPodAutoscalerSpec struct {
 	// more information about how each type of metric must respond.
 	// +optional
 	Metrics []MetricSpec
+
+	// behavior configures the scaling behavior of the target
+	// in both Up and Down directions (scaleUp and scaleDown fields respectively).
+	// If not set, the default HPAScalingRules for scale up and scale down are used.
+	// +optional
+	Behavior *HorizontalPodAutoscalerBehavior
+}
+
+// HorizontalPodAutoscalerBehavior configures a scaling behavior for Up and Down direction
+// (scaleUp and scaleDown fields respectively).
+type HorizontalPodAutoscalerBehavior struct {
+	// scaleUp is scaling policy for scaling Up.
+	// If not set, the default value is used:
+	// - The first policy is to increase no more than 4 pods per 60 seconds.
+	// - The second policy limit is to double the number of pods per 60 seconds.
+	// - The policy that provides the highest change is picked.
+	// - No stabilization is used.
+	// +optional
+	ScaleUp *HPAScalingRules
+	// scaleDown is scaling policy for scaling Down.
+	// if not set, the default value is used:
+	// - The only policy specified is to allow to scale down to 0 pods.
```

Comment:
Really?  We scale to 0 by default?

Reply:
Well, we don't limit it somehow.
But you are right, I had an incorrect comment, we allow scaling to minReplicas, not to 0.

Reply:
Changed the comment.